frontend_app_workspace/src/container/PopupCreateWorkspace.jsx

```diff
@@ -192,7 +192,11 @@ export class PopupCreateWorkspace extends React.Component {
     switch (fetchGetAllowedSpaceTypes.apiResponse.status) {
       case 200: {
         const apiTypeList = fetchGetAllowedSpaceTypes.body.items
-        this.setState({ allowedTypes: SPACE_TYPE_LIST.filter(type => apiTypeList.some(apiType => apiType === type.slug)) })
+        const allowedTypesList = SPACE_TYPE_LIST.reverse().filter(type => apiTypeList.some(apiType => apiType === type.slug))
```

Comment:
It looks like to me that ideally, the code should not depend on the order of elements in `SPACE_TYPE_LIST`.

It's fine for me to keep `reverse()` as-is anyway for now, but can you put a comment at the location where `SPACE_TYPE_LIST` is defined to say that the order matters with a mention like `please see frontend_app_workspace/src/container/PopupCreateWorkspace.jsx`?

Reply:
Updated